### PR TITLE
Update changelog for 5.2.0

### PR DESCRIPTION
**Pull request by owenatgov**
What it says on the tin. Taken from [release notes doc](https://docs.google.com/document/d/1LG5pgElv6rHLF40g3hUOD96rnRsyngAsUYyV6rEOmWQ/edit) (internal only). Has been 2i'd.

Closes https://github.com/alphagov/govuk-frontend/issues/4698

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,15 @@
...
 
-## Unreleased
+## 5.2.0 (feature release)
 
-### New features
+In this release, we’ve adjusted our responsive type scale, which is available behind a feature flag. The type scale change is to make text easier to read on smaller screens. We’ve also deprecated the `useTudorCrown` parameter.
+
+To install this version with npm, run `npm install govuk-frontend@5.2.0`. You can also find more information about [how to stay up to date](https://frontend.design-system.service.gov.uk/staying-up-to-date/#updating-to-the-latest-version) in our documentation.
+
+## New features
```
Comment (36degrees) [inline CHANGELOG.md:11]: The heading levels are wrong from this point onwards – everything needs to be dropped a level.

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,23 +23,25 @@ We've made the following adjustments to our responsive type scale:
 - point 36 remains as 27px on large screens
 - point 36 now returns 27px on small screens instead of 24px and has a line height 30px instead of 25px
 
-To enable these changes, set `$govuk-new-typography-scale` to `true` before you import GOV.UK Frontend in your Sass:
+To enable these changes, set the feature flag variable `$govuk-new-typography-scale` to true before you import GOV.UK Frontend in your Sass files:
```
Comment (36degrees) [inline CHANGELOG.md:26]: Not sure if this was an intentional change – think we would normally use inline code for `true` here?

```suggestion
To enable these changes, set the feature flag variable `$govuk-new-typography-scale` to `true` before you import GOV.UK Frontend in your Sass files:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,23 +23,25 @@ We've made the following adjustments to our responsive type scale:
...
 
-#### Insert custom HTML into component form group wrappers
+This change was introduced in [pull request #2421: Adjust the responsive type scale](https://github.com/alphagov/govuk-frontend/pull/2421)
+
+### Insert custom HTML into component form group wrappers
 
 You can now insert custom HTML into form group wrappers for all components with form fields.
 
-```njk
+```
```
Comment (36degrees) [inline CHANGELOG.md:44]: Can we keep the syntax highlighting here?

```suggestion
```njk
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,23 +23,25 @@ We've made the following adjustments to our responsive type scale:
 - point 36 remains as 27px on large screens
 - point 36 now returns 27px on small screens instead of 24px and has a line height 30px instead of 25px
 
-To enable these changes, set `$govuk-new-typography-scale` to `true` before you import GOV.UK Frontend in your Sass:
+To enable these changes, set the feature flag variable `$govuk-new-typography-scale` to true before you import GOV.UK Frontend in your Sass files:
 
 ```
```
Comment (36degrees) [inline CHANGELOG.md:28]: Can we add syntax highlighting for this?

```suggestion
```scss
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -48,17 +54,19 @@ govukInput({
 })
 ```
 
-This change was introduced in [pull request #4567: Add `beforeInput(s)` and `beforeInput(s)` options to form groups](https://github.com/alphagov/govuk-frontend/pull/4567).
+This change was introduced in pull request #4567: Add beforeInput(s) and beforeInput(s) options to form groups.
```
Comment (36degrees) [inline CHANGELOG.md:57]: The link and inline codes have gone.

```suggestion
This change was introduced in [pull request #4567: Add `beforeInput(s)` and `beforeInput(s)` options to form groups](https://github.com/alphagov/govuk-frontend/pull/4567).
```

**Review by domoscargin — APPROVED** [review]

**Pull request merged**